trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -120,15 +162,35 @@ def VLM(conditions,settings,geometry):
         x_m = x_cg
         z_m = z_cg
         
+    # unpack conditions--------------------------------------------------------------
     aoa  = conditions.aerodynamics.angle_of_attack   # angle of attack  
     mach = conditions.freestream.mach_number         # mach number
     ones = np.atleast_2d(np.ones_like(mach)) 
     len_mach = len(mach)
+    
+    #For angular values, VORLAX uses degrees by default to radians via DTR (degrees to rads). 
+    #SUAVE uses radians and its Units system. All algular variables will be in radians or var*Units.degrees
+    PSI       = conditions.aerodynamics.side_slip_angle     
+    PITCHQ    = conditions.stability.dynamic.pitch_rate              
+    ROLLQ     = conditions.stability.dynamic.roll_rate             
+    YAWQ      = conditions.stability.dynamic.yaw_rate 
+    VINF      = conditions.freestream.velocity    
+       
+    #freestream 0 velocity safeguard
+    if not conditions.freestream.velocity.all():
+        if settings.use_surrogate:
+            velocity                       = conditions.freestream.velocity
+            velocity[velocity==0]          = np.ones(len(velocity[velocity==0])) * 1e-6
+            conditions.freestream.velocity = velocity
+        else:
+            raise AssertionError("VLM requires that conditions.freestream.velocity be specified and non-zero")    
 
-    # generate vortex distribution 
+    # ---------------------------------------------------------------------------------------
+    # STEPS 1-9: Generate Panelization and Vortex Distribution
+    # ------------------ --------------------------------------------------------------------    
+    # generate vortex distribution (VLM steps 1-9)
     VD   = generate_vortex_distribution(geometry,settings)  
     
-    
     # Unpack vortex distribution
     n_cp         = VD.n_cp
     n_w          = VD.n_w    
```

Comment:
VD.n_w is not used anymore in this script

Reply:
removed